src/NzbDrone.Core/HealthCheck/Checks/MountCheck.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Linq;
+using NzbDrone.Common.Disk;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.HealthCheck.Checks
+{
+    public class MountCheck : HealthCheckBase
+    {
+        private readonly IDiskProvider _diskProvider;
+        private readonly ISeriesService _seriesService;
+
+        public MountCheck(IDiskProvider diskProvider, ISeriesService seriesService)
+        {
+            _diskProvider = diskProvider;
+            _seriesService = seriesService;
+        }
+
+        public override HealthCheck Check()
+        {
+            var mounts = _seriesService.GetAllSeries()
+                                       .Select(series => _diskProvider.GetMount(series.Path))
+                                       .DistinctBy(m => m.RootDirectory)
```

Comment:
Already discussed it on IRC, we can't use Distinct coz we need mono to do the SymLinkResolver per series path to get the right mount. So we opted to just add a comment to explain why. It's not like this code gets call often.